backend/app/controllers/schedules_controller.ts

```diff
@@ -1,35 +1,52 @@
 import type { HttpContext } from '@adonisjs/core/http'
 import Schedule from '#models/schedule'
 import { createScheduleValidator, updateScheduleValidator } from '#validators/schedule'
+
 export default class SchedulesController {
   /**
    * Display a list of user schedules
    */
-  async index({ params }: HttpContext) {
-    const userId = params.user_id
-
-    if (userId) {
-      return Schedule.query().where('userId', userId)
+  async index({ auth }: HttpContext) {
+    const userId = auth.user?.id
```

Comment:
potrzebujemy to skoro chyba middleware robiles? przynajmniej marcin pokazywal

Reply:
teraz zerknąłem jeszcze raz na to, to ci chodzi o to z params.user_id? bo jesli tak to jest przeciez usuniete xd i korzystam tylko z id z auth'a

Reply:
czy ci się źle zaznaczylo i chodzi ci o to co jest nizej?`if (!userId) {
      return { message: 'User not authenticated.' }`

Reply:
tak tak